lavocado/test.py

```diff
@@ -54,34 +55,39 @@ def connect(self):
                    + self.defaults.LIBVIRT_URI)
             self.cancel(msg)
 
-    # TODO: Use @fail_on decorator
-    def create_domain(self, arguments=None, fail=False):
+    def create_domain(self, arguments=None):
```

Comment:
I wonder if, with the removal of `fail`, and being the only parameter, if `arguments` should be turned into keyword args.

Reply:
TBH, I don't like much kwargs for this use. If we decide to go on that way, we could convert this into arguments to make it explicit.

Reply:
The question is how many arguments would we need to support?  Keyword args are specially suited to avoid huge function/method signatures wrt parameters.

Reply:
`arguments` is a dict with values for parsing the template. In that way we could also grow without messing with the method signature and keep a separation between method's arguments and template parse arguments.